core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -127,6 +129,7 @@ public class TableTestBase {
           .withFileSizeInBytes(10)
           .withPartitionPath("data_bucket=3") // easy way to set partition data for now
           .withRecordCount(1)
+          .withSplitOffsets(ImmutableList.of(4L, 10_000_000L, 20_000_000L))
```

Comment:
Minor comment but can we make the splits unique? Just want to make sure things will break in an obvious way if they are not matching

Reply:
Sure, I made this change